.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be added to .gitignore probably?

Reply:
yes, I will remove this file when I update the commit